apps/aens/test/aens_SUITE.erl

```diff
@@ -560,3 +578,122 @@ do_prune_until(N1, N1, OTree) ->
     aens_state_tree:prune(N1, OTree);
 do_prune_until(N1, N2, OTree) ->
     do_prune_until(N1 + 1, N2, aens_state_tree:prune(N1, OTree)).
+
+
+%%%===================================================================
+%%% Subdomain
+%%%===================================================================
+
+-define(TOPNAME, <<"姆.test"/utf8>>).
+-define(SNAME1, <<"斯1.姆.test"/utf8>>).
+-define(SNAME2, <<"姆2.姆.test"/utf8>>).
+-define(SNAME21, <<"姆2.斯1.姆.test"/utf8>>).
+-define(SNAME54321, <<"詹5.斯4.詹3.姆2.斯1.姆.test"/utf8>>).
+
+-define(SNAME321, <<"詹3.姆2.斯1.姆.test"/utf8>>).
+-define(SNAME4321, <<"斯4.詹3.姆2.斯1.姆.test"/utf8>>).
+
+subname(_Cfg) ->
```

Comment:
Should not this test be running only for `ct-lima` CI job (or equivalent from point of view of consensus) and later? Is it conditional somewhere and I am not noticing it?